### PR TITLE
Capitalise Kedro-Viz in the "Visualize layers" section

### LINKED ISSUE #1896: Capitalise Kedro-Viz in documentation
## Description
This is hopefully a quick win exercise. We want you to fix a typo in our documentation where we should have capitalised Kedro-Viz but forgot to. 

To find and fix the error: 
1. Head to the [Visualise Pipelines section in our documentation](https://github.com/kedro-org/kedro/blob/main/docs/source/tutorial/visualise_pipeline.md#visualise-layers)
2. Search for "kedro-viz", the affected area is circled in blue
3. Replace with "Kedro-Viz"

Screenshot 2022-10-04 at 15 43 28

## Context
You're helping us keep our style guidelines in tact.

## Instructions on how to contribute

Checkout our [guidelines by @stichbury](https://kedro.readthedocs.io/en/stable/contribution/documentation_contributor_guidelines.html) and [head to Slack](https://join.slack.com/t/kedro-org/shared_invite/zt-1eicp0iw6-nkBvDlfAYb1AUJV7DgBIvw) if you have any questions.

### PR DESCRIPTION
**Pull request by yash6318**
## Description

This PR was created as it fixes #1896 and it would be a start of Hactoberfest'22 for me.

## Development notes

`kedro-viz` was capitalised to `Kedro-Viz` in Visualize Layers section in the documentation as requested.

## Checklist

- [x] Read the [contributing](https://github.com/kedro-org/kedro/blob/main/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [x] Updated the documentation to reflect the code changes
- [ ] Added a description of this change in the [`RELEASE.md`](https://github.com/kedro-org/kedro/blob/main/RELEASE.md) file
- [x] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro/pull/1899"></a>

### CODE REVIEW

**Review by MerelTheisenQB — APPROVED** [review]

**Comment by yash6318** [comment]
> Looks great! While you're at it, can you also fix the "Kedro viz" in `docs/source/tutorial/set_up_experiment_tracking.md` in the same PR? I just did a quick search through the repo, but feel free to also see if you catch any other instances.

This is done! Moreover, no other occurences of "Kedro viz" were found. 
(Also, If you want other changes too that are similar to plz ask! )

File: docs/source/tutorial/set_up_experiment_tracking.md [hunk 1]
```diff
@@ -225,7 +225,7 @@ After running the pipeline with `kedro run`, the plot will be saved and you will
 ![](../meta/images/expand-plot-comparison-view.gif)
 
 
-Read more about [creating plots and visualising them in Kedro viz in the visualise pipeline section.](../tutorial/visualise_pipeline.md#visualise-charts-in-kedro-viz)
+Read more about [creating plots and visualising them in Kedro Viz in the visualise pipeline section.](../tutorial/visualise_pipeline.md#visualise-charts-in-kedro-viz)
```
Comment (deepyaman) [inline docs/source/tutorial/set_up_experiment_tracking.md:228]:
```suggestion
Read more about [creating plots and visualising them in Kedro-Viz in the visualise pipeline section.](../tutorial/visualise_pipeline.md#visualise-charts-in-kedro-viz)
```
Minor change, and we're good to merge!

Comment (yash6318) [reply docs/source/tutorial/set_up_experiment_tracking.md]: Done!

**Review by deepyaman — APPROVED** [review]
@yash6318 Thanks for making the changes, and congrats on your first contribution!

**Comment by yash6318** [comment]
Thankss!

**Pull request merged**